.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["eslint:recommended", "plugin:import/recommended"],
-  "ignorePatterns": ["dist", "node_modules"],
+  "ignorePatterns": ["dist", "node_modules", "*.ts"],
```

Comment:
Is this intentional?

Reply:
Yes, we don't have typescript enabled and eslint gets confused about it.

Reply:
Got it, makes sense 👍